cmd/openshift-install/main.go

```diff
@@ -28,8 +29,10 @@ func main() {
 	// to log anything.
 	var fs flag.FlagSet
 	klog.InitFlags(&fs)
+	klogv2.InitFlags(&fs)
```

Comment:
I think using the same flag set for both v1 and v2 is causing this failure https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_installer/4266/pull-ci-openshift-installer-master-e2e-aws/1316299561756528640#1:build-log.txt%3A129